components/tree-grid/__examples__/default.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import TreeGrid from '~/components/tree-grid';
+import IconSettings from '~/components/icon-settings';
+
+const data = {
```

Comment:
If there were no nested rows, then a dev would use a DataTable. I don't think we need this example and base should be nested.